src/core/path.c

```diff
@@ -26,6 +26,7 @@ QuicPathInitialize(
     )
 {
     CxPlatZeroMemory(Path, sizeof(QUIC_PATH));
+    CxPlatInitializeRoute(&Path->Route);
```

Comment:
Redundant as it's zero'd out above.

Reply:
The thing is you now have an initializer that only zeros out memory and the actual initializing code is all over the place. IMO, it's making the code even more confusing.

Reply:
got rid of wrapper.